index.rkt

```diff
@@ -350,7 +350,7 @@ Finally we use @racket[datum->syntax] to convert this back to
 @racket[syntax]:
 
 @i[
-(datum->syntax #f '(values "i" "am" "backwards"))
+(datum->syntax stx '(values "i" "am" "backwards"))
 ]
```

Comment:
I think this needs to remain `#f` -- because `stx` isn't defined and it would error if people are trying it while they read along?